lib/atomic/accounts/user.ex

```diff
@@ -15,8 +15,8 @@ defmodule Atomic.Accounts.User do
     field :confirmed_at, :naive_datetime
 
     field :role, Ecto.Enum, values: @roles
-
     has_many :enrollments, Enrollment
+    has_one :collaborator, Collaborator, on_replace: :delete_if_exists, on_delete: :delete_all
```

Comment:
```suggestion
    has_many :collaborator, Collaborator, on_replace: :delete_if_exists, on_delete: :delete_all
```
A user can be a collaborator of multiple departments

Reply:
collaborators*

Reply:
I updated to a many_to_many relationship